packages/calcite-components/src/components/action-bar/action-bar.scss

```diff
@@ -3,59 +3,53 @@
  *
  * These properties can be overridden using the component's tag as selector.
  *
- * @prop --calcite-action-bar-expanded-max-width: When `layout` is `"vertical"`, specifies the expanded max width of the component.
+ * @prop --calcite-action-bar-expanded-max-width: Specifies the maximum width of the component when it's `layout` is `"vertical"`.
+ * @prop --calcite-action-bar-items-space: Specifies the space between slotted components in the component.
```

Comment:
Do we need this prop? From a design standpoint, not sure what the reasoning for having this capability is? cc @ashetland @SkyeSeitz

Reply:
Agreed not needed imo